libs/island-ui/core/src/lib/Tag/Tag.tsx

```diff
@@ -70,7 +67,7 @@ export const Tag = forwardRef<HTMLButtonElement & HTMLAnchorElement, TagProps>(
       </Text>
     )
 
-    if (label) {
+    if (outlined) {
```

Comment:
Just a though for another PR.. I'm not sure we want to define clickable or not by the outlined value. Would it make sense to use a button if the `onClick` function is present? something like what's happening with the `href` attribute? so if there is no `href` and no `onClick` then we would return this span?